test.c

```diff
@@ -3664,6 +3664,30 @@ test_header_cr_no_lf_error (int req)
   abort();
 }
 
+void
+test_no_overflow_parse_url (void)
+{
+  int rv;
+  struct http_parser_url u;
+
+  http_parser_url_init(&u);
+  rv = http_parser_parse_url("http://example.com:8001", 22, 0, &u);
+
+  if (rv != 0) {
+    fprintf(stderr,
+            "\n*** test_no_overflow_parse_url invalid return value =%d\n",
```

Comment:
Teeny tiny nit: no space before `=`, you don't have one below either.

Reply:
Ack.